src/joLinearFunctionConstructors/joJRM.jl

```diff
@@ -0,0 +1,73 @@
+# joJRM operator
+
+## helper module
+module joJRM_etc
+
+    function apply_fwd(n, L, z; γ=1f0)
+        return view(z, n+1:length(z)) .+ 1f0/γ * repeat(view(z, 1:n), L)    # get original components
+    end 
+
+    function apply_adj(L, x; γ=1f0)
+        x0 = 1f0/γ * sum(reshape(view(x,:), :, L), dims=2)   # weighted sum on common component
+        return vcat(x0, x)
+    end
+
+end
+using .joJRM_etc
+
+export joJRM
+"""
+    julia> op = joJRM(ops;[name=...])
```

Comment:
This is not very descriptive, a generic user will have zero idea what this is.

And again, JRM is the inversion method, this is just a block operator not an algorithm.  It's not said anywhere what JRM even means